lib/instrumentation/modules/http2.js

```diff
@@ -179,6 +184,14 @@ module.exports = function (http2, agent, { enabled }) {
           url
         })
 
+        if (typeof statusCode !== 'undefined') {
+          if (statusCode >= 400) {
+            span.outcome = 'failure'
+          } else {
+            span.outcome = 'success'
+          }
+        }
+
```

Comment:
Here I see that during our handling of calls the the `request` method of of objects returned by http.connect we're examining the HTTP2 response code and setting the outcome property to `failure` if the status code is greater than or equal to 400.  This behavior matches the spec.